types/custom-functions-runtime/index.d.ts

```diff
@@ -48,6 +48,11 @@ declare namespace CustomFunctions {
      * Provides information about the invocation of a custom function.
      */
     interface Invocation {
+        /**
+         * The name of this function.
+         * [Api set: CustomFunctionsRuntime 1.1]
+         */
+        functionName: string;
```

Comment:
@alison-mk  Doesn't this need to be `functionName?: string` since it may be undefined.

Reply:
Thanks for the review! Discussing on [internal work item](https://office.visualstudio.com/OC/_workitems/edit/5319473).